plugins/remote-config/frontend/public/javascripts/countly.views.js

```diff
@@ -13,51 +18,247 @@
+                                <div v-if="tag.key == selectedColorKey" @click="click" class="cly-vue-remote-config-conditions-drawer__color-tag cly-vue-remote-config-conditions-drawer__color-tag__selected bu-is-flex bu-is-align-items-center bu-is-justify-content-center" :style="{backgroundColor: tag.colorCode}" :data-color="tag.key">\
+                                    <i class="ion-checkmark cly-vue-remote-config-conditions-drawer__checkmark"></i>\
+                                </div>\
+                                <div v-else @click="click" class="cly-vue-remote-config-conditions-drawer__color-tag bu-is-flex bu-is-align-items-center bu-is-justify-content-center" :style="{backgroundColor: tag.colorCode}" :data-color="tag.key"></div>\
+                                </div>\
+                    </div>'
+    });
+    var JsonEditor = countlyVue.views.create({
+        template: CV.T("/remote-config/templates/json-editor.html"),
+        data: function() {
+            return {
+                jsonerror: "",
+                jsonstr: ""
+            };
+        },
+        computed: {
+            centerDialogVisible: function() {
+                return this.$store.getters["countlyRemoteConfig/parameters/showJsonEditor"];
+            }
+        },
+        methods: {
+            validateJson: function() {
+                this.jsonerror = "";
+                try {
+                    // try to parse
+                    if (this.jsonstr) {
+                        JSON.parse(this.jsonstr);
+                    }
+                }
+                catch (e) {
+                    this.jsonerror = JSON.stringify(e.message);
+                }
+            },
+            prettyFormat: function() {
+                // reset error
+                var jsonValue = "";
+                try {
+                    // try to parse
+                    jsonValue = JSON.parse(this.jsonstr);
+                    this.jsonstr = JSON.stringify(jsonValue, null, 2);
+                }
+                catch (e) {
+                    // Do nothing
+                }
+            },
+            submit: function() {
+                this.$store.dispatch("countlyRemoteConfig/parameters/showJsonEditor", false);
+            },
+            cancel: function() {
+                this.$store.dispatch("countlyRemoteConfig/parameters/showJsonEditor", false);
+            }
+        }
+    });
     var ConditionsDrawer = countlyVue.views.create({
         template: CV.T("/remote-config/templates/conditions-drawer.html"),
+        components: {
+            "color-tag": ColorTag
+        },
         props: {
             controls: {
                 type: Object
             }
         },
+        computed: {
+            showSeedValue: function() {
+                if (!_.isEmpty(this.managedPropertySegmentation.queryText) && this.managedPropertySegmentation.queryText.includes("Random Percentile")) {
```

Comment:
please use some id check in the query object instead of checking for the Random percentile in the queryText. It won't work as expected if the string is localized.
I am sure random percentile must have some id or something.

Reply:
It is not possile with the current object. But can did give the solution for this yestaerday, havent commited it yet. Will update the pr :)